### PR TITLE
[bn] Translate Update

### PR DESCRIPTION
**Pull request by arman-arif**
Latest translate update

### CODE REVIEW

File: locales/bn/json.json [hunk 1]
```diff
@@ -573,11 +573,11 @@
     "Sri Lanka": "শ্রীলঙ্কা",
     "Standalone Actions": "স্বতন্ত্র অ্যাকশন",
     "Start Polling": "নির্বাচনের আরম্ভ করুন",
-    "Start polling": "Start polling",
+    "Start polling": "ভোটগ্রহণ শুরু করুন",
     "State / County": "প্রদেশ / দেশ",
     "Stop Impersonating": "ছদ্মবেশী করা বন্ধ করুন",
     "Stop Polling": "ভোটদান বন্ধ করুন",
-    "Stop polling": "Stop polling",
+    "Stop polling": "ভোটগ্রহণ বন্ধ করুন",
```
Comment (andrey-helldar) [inline locales/bn/json.json:580]: Just a clarification question: is it correct that the translation into Bengali differs from the translation in the `Stop Polling` key one line up?

Comment (arman-arif) [reply locales/bn/json.json:580]: It's all the same

Comment (andrey-helldar) [reply locales/bn/json.json:580]: Ok, thanks

**Review by andrey-helldar — APPROVED** [review]

**Comment by andrey-helldar** [comment]
তোমাকে ধন্যবাদ

File: locales/bn/json.json (hunk 1 above)
Comment (arman-arif) [reply locales/bn/json.json:580]: Should I create a new PR?

Comment (andrey-helldar) [reply locales/bn/json.json:580]: Yes, I already got this one :)

**Pull request merged**